src/Core/Client/Adapter/Curl.php

```diff
@@ -107,6 +107,14 @@ public function createHandle(Request $request, Endpoint $endpoint)
             curl_setopt($handler, CURLOPT_USERPWD, $authData['username'].':'.$authData['password']);
             curl_setopt($handler, CURLOPT_HTTPAUTH, CURLAUTH_BASIC);
         }
+        elseif (!isset($options['headers']) || !array_key_exists('Authorization', $options['headers'])) {
```

Comment:
```suggestion
        elseif (!isset($options['headers']) || !\array_key_exists('Authorization', $options['headers'])) {
```

In keeping with https://github.com/FriendsOfPHP/PHP-CS-Fixer/blob/866715801ac121a0a4cde6fdb9d9a3dd6873b606/src/Fixer/FunctionNotation/NativeFunctionInvocationFixer.php#L347